test-plone-5.1.x.cfg

```diff
@@ -0,0 +1,21 @@
+[buildout]
+extends =
+    http://kgs.4teamwork.ch/sources.cfg
+    https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/test-plone-5.1.x.cfg
+
+package-name = ftw.globalstatusmessage
+
+extensions += mr.developer
+
+auto-checkout =
+    ftw.publisher.core
+    ftw.publisher.receiver
+
+[branches]
+ftw.publisher.core = ne/plone-5.1
+ftw.publisher.receiver = plone-5.1.x
```

Comment:
🚧 I'd wait for releases here and remove it before merging, so that we start with a clean state.